fil-proofs-tooling/src/metadata.rs

```diff
@@ -109,6 +110,33 @@ impl SystemMetadata {
             memory_total_bytes: memory.total().get::<uom::si::information::byte>(),
         })
     }
+
+    #[cfg(not(target_arch = "x86_64"))]
```

Comment:
I personally prefer keeping the `cfg()` differences as small as possible. We could create a methos that returns the host, memory and the cpu_logical/physical that is shared between the `new()`s. This would also make it more apparent what the difference between the cfgs is. Though it''s a matter of test, hence just a comment and nothing that needs to change/blocking a merge.

Reply:
I agree this could be nicer, but I mostly wanted to get this to compile

Reply:
@keyvank